src/offset/local/stub.rs

```diff
@@ -8,218 +8,12 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-use std::time::{SystemTime, UNIX_EPOCH};
+use crate::{FixedOffset, LocalResult, NaiveDateTime};
```

Comment:
I think I'd prefer to keep the non-WASM and `wasmbind` code in the same module. Or maybe both `stub` and `wasmbind` can become inline modules? In either case, please think about the optimally reviewable commit sequence for making changes like this.

Reply:
Inline modules are a good idea 👍.

> In either case, please think about the optimally reviewable commit sequence for making changes like this.

I made an effort.

Reply:
Looks good, thanks!